### PR TITLE
(Chore) Added delivery leads guide for arranging pen tests

### PR DESCRIPTION
**Pull request by olivia-work**

### CODE REVIEW

File: guides/delivery/delivery-lead-guide-pen-tests.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Delivery leads guide for arranging pen tests
+
+When a penetration test is arranged by a client on a site which is hosted by
```
Comment (lozette) [inline guides/delivery/delivery-lead-guide-pen-tests.md]: This sentence feels like a run-on sentence. Maybe add a couple of commas: 

`When a penetration test is arranged by a client on a site which is hosted by dxw digital, the delivery leads must find out the scope of the test and raise a ticket in Zendesk, as the Ops team will need to notify AWS as part of our contract with them.`

Comment (staceydxw) [reply guides/delivery/delivery-lead-guide-pen-tests.md]: Yes that looks better

**Comment by staceydxw** [comment]
Bit confused now about whether I can make the changes and merge or whether someone else needs to do it. Let me know if I can help :)

File: guides/delivery/delivery-lead-guide-pen-tests.md (hunk 1 above)
Comment (CristinaRO) [reply guides/delivery/delivery-lead-guide-pen-tests.md]: @olivia-work Given that this is your PR, would you mind one of us making the above changes, or do you want to make them yourself?

**Comment by olivia-work** [comment]
This document is no longer required because AWS has announced that we no longer need to notify them when a pen test is taking place.

**Pull request closed without merging**